releasenotes/notes/add-backend-sampler-v2-5e40135781eebc7f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    The implementation :class:`~.BackendSamplerV2` of :class:`~.BaseSamplerV2` was added.
+    This requires :class:`~.BackendV2` object that supports ``memory`` option to computes bitstrings.
```

Comment:
Do you mind adding an example like https://github.com/Qiskit/qiskit/pull/11928#issuecomment-1977390888 ?